src/main/kotlin/app/utils/UiHelper.kt

```diff
@@ -4,8 +4,15 @@
 package app.utils
 
 object UiHelper {
-    fun confirm(message: String): Boolean {
-        println("$message [y/n]")
-        return (readLine() ?: "").toLowerCase() == "y"
+    fun confirm(message: String, default: Boolean): Boolean {
+        val yes = "y"
```

Comment:
This could be simplified to:
```
val yes = "Y" if defaultIsYes else "y";
val no = "N" if !defaultIsYes else "n";
println("$message [$yes/$no]);
```